spec/cf_app_discovery/parser_spec.rb

```diff
@@ -15,7 +15,8 @@
           state: "STARTED",
           detected_start_command: "./bin/paas-metric-exporter",
         },
-        route: "route-1",
+        hostname: "route-1",
```

Comment:
Small but wonder if `host-1` etc would be a bit clearer so we don't introduce confusion between the hostname and routes?

Reply:
i wondered about this. i don't think it's clear what the right answer is here because the hostname is part of the route. therefore i erred on the side of making fewer code changes